### PR TITLE
Documentation update for llama-v2

### PR DESCRIPTION
**Pull request by yunfeng-scale**

### CODE REVIEW

File: docs/model_zoo.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Scale hosts the following models in the LLM Engine Model Zoo:
 | Model Name            | Inference APIs Available | Fine-tuning APIs Available |
 | --------------------- | ------------------------ | -------------------------- |
 | `llama-7b`            | ✅                       | ✅                         |
+| `llama-7b-2`          | ✅                       | ✅                         |
```
Comment (seanshi-scale) [inline docs/model_zoo.md:8]: should this be `llama-2-7b`? on https://huggingface.co/meta-llama/Llama-2-7b it's that at least

Comment (yixu34) [reply docs/model_zoo.md:8]: Ooooh good catch

Comment (yunfeng-scale) [reply docs/model_zoo.md:8]: oh that's right

**Review by yixu34 — APPROVED** [review]

**Comment by yunfeng-scale** [comment]
[sc-783347]

**Pull request merged**